### PR TITLE
Bootstrapify pages

### PR DESCRIPTION
**Pull request by jml**
Makes the pages much prettier. I haven't done anything with layout, but I'm happy to take concrete suggestions. 

Haven't tested the Docker image, due to not being on Linux, and forgetting the Stack command to build from within Docker.

### CODE REVIEW

**Comment by awh** [comment]
Needs screenshot

**Comment by jml** [comment]
screen shot 2018-06-25 at 15 58 22

Note I haven't changed the HTML in any way. Suggestions welcome.

**Comment by jml** [comment]
You asked for jumbotron.

## Home page

screen shot 2018-07-02 at 12 17 06

## Images

screen shot 2018-07-02 at 12 17 12

## Revisions

screen shot 2018-07-02 at 12 16 56

## Changelog

screen shot 2018-07-02 at 12 17 21

I hope you're happy.

I've changed the lucid import to be `import Lucid`, bringing all the symbols into the API module's namespace. While I don't like this practice in general, I think it makes the HTML templates easier on the eye.

File: src/CompareRevisions/API.hs [hunk 1]
```diff
@@ -215,23 +213,32 @@ data RootPage = RootPage URI (Map Config.EnvironmentName FilePath) deriving (Eq,
...
-      -- servant 0.14 would allow us to use safeLink', which would let us build a helper to reduce duplication.
-      li_ $ a_ [href_ (show (linkURI (safeLink api imagesEndpoint)))] "Images"
-      li_ $ a_ [href_ (show (linkURI (safeLink api revisionsEndpoint)))] "Revisions"
-    h2_ "Within environments"
-    ul_ $ sequence_ [ li_ $ a_ [href_ (show (linkURI (safeLink api changelogEndpoint env Nothing)))] (toHtml env)
-                      | env <- Map.keys envs ]
-    h2_ "Ops"
-    ul_ $
-      li_ $ a_ [href_ (show externalURL <> "/metrics")] (code_ "metrics")
+    div_ [class_ "jumbotron"] $ do
```
Comment (awh) [inline src/CompareRevisions/API.hs]: All hail the jumbotron!

**Review by awh — APPROVED** [review]
* Unqualified Lucid import :heart: 
* A few places mention things can be improved with servant 0.14 - any reason why not to use it now?
* `Down` is an odd name

**Comment by jml** [comment]
> A few places mention things can be improved with servant 0.14 - any reason why not to use it now?

I was already concerned with the amount of time I'd spent on the PR, and the cost of review. Also, when I wrote the comments, servant 0.14 was not released. I'll happily do a follow-up PR that switches to 0.14.

**Pull request merged**